packages/telemetry/src/base/types/event-name.ts

```diff
@@ -0,0 +1,57 @@
+export enum EventName {
+    Test = 'test',
+    TELEMETRY_SETTINGS_INIT_FAILED = 'TELEMETRY_SETTINGS_INIT_FAILED',
+    DISABLE_TELEMETRY = 'DISABLE_TELEMETRY',
+    GD_GUIDE_ACTION = 'GUIDE_ACTION',
+    GD_PANEL_LOAD = 'PANEL_LOAD',
+    GD_LAUNCH = 'LAUNCH_GD',
+    GD_SIDE_BY_SIDE = 'GD_SIDE_BY_SIDE',
+    GD_WIZARD = 'GD_WIZARD',
+    GD_DESCMODE = 'GD_DESCMODE',
+    GD_PROJECTVIEW = 'GD_PROJECTVIEW',
+    // Unique to Application Modeler
+    PAGEMAP = 'PAGEMAP',
+    APP_ACTIONS = 'ACTIONS',
+    PAGE_EDITOR = 'PAGE_EDITOR',
+    PAGEMAP_STARTUP_TIME = 'PAGEMAP_STARTUP_TIME',
+    APPLICATION_PREVIEW = 'APPLICATION_PREVIEW',
+    APP_INFO_COMMAND_STARTED = 'APP_INFO_COMMAND_STARTED',
+    APP_INFO_LINK_CLICKED = 'APP_INFO_LINK_CLICKED',
+    APP_INFO_STARTUP_TIME = 'APP_INFO_STARTUP_TIME',
+    // Unique to Service Modeler
+    SERVICE_MODELER_EVENT = 'SERVICE_MODELER_EVENT',
+    // Unique to Annotation File Mgr
+    ANNOTATION_FILE_MANAGER_EVENT = 'ANNOTATION_FILE_MANAGER_EVENT',
+    // Unique to Annotation Modeler
+    ANNOTATION_LSP_XML_LOAD = 'ANNOTATION_LSP_XML_LOAD',
+    ANNOTATION_LSP_USAGE_TERM = 'ANNOTATION_LSP_USAGE_TERM',
+    ANNOTATION_LSP_CODE_COMPLETION = 'ANNOTATION_LSP_CODE_COMPLETION',
+    ANNOTATION_LSP_CODE_COMPL_LOAD = 'ANNOTATION_LSP_CODE_COMPL_LOAD',
+    // Unique to Migration
+    MIGRATION_ACTIVATED = 'MIGRATION_ACTIVATED',
+    MIGRATION_BACKEND_LOAD = 'MIGRATION_BACKEND_LOAD',
+    MIGRATION_REFRESH_BACKEND_LOAD = 'MIGRATION_REFRESH_BACKEND_LOAD',
+    MIGRATION_BACK_BACKEND_LOAD = 'MIGRATION_BACK_BACKEND_LOAD',
+    MIGRATION_COMPLETED = 'MIGRATION_COMPLETED',
+    MIGRATION_SUCCESS = 'MIGRATION_SUCCESS',
+    MIGRATION_FAILED = 'MIGRATION_FAILED',
+    MIGRATION_SHOW_INFO_PAGE = 'MIGRATION_SHOW_INFO_PAGE',
+    MIGRATION_ADD_PROJECT = 'MIGRATION_ADD_PROJECT',
+    DEPLOY_CONFIG = 'DEPLOY_CONFIG',
+    DEPLOY = 'DEPLOY',
+    DEPLOY_FAIL = 'DEPLOY_FAIL',
+    UNDEPLOY = 'UNDEPLOY',
+    UNDEPLOY_FAIL = 'UNDEPLOY_FAIL',
+    // Unique to Data Editor
+    DATA_EDITOR_EVENT = 'DATA_EDITOR_EVENT',
+    // Unique to Control Property Editor
+    CPE_EVENT = 'CONTROL_PROPERTY_EDITOR_EVENT',
+    // Unique to System View
+    SYSTEM_DETAILS_VIEW_EVENT = 'SYSTEM_DETAILS_VIEW_EVENT',
+    // App generator
+    SERVICE_INQUIRER_BAS_SUCCESS = 'SERVICE_INQUIRER_BAS_SUCCESS',
+    SERVICE_INQUIRER_BAS_ERROR = 'SERVICE_INQUIRER_BAS_ERROR',
+    GA_LINK_CREATED = 'GA_LINK_CREATED',
+    // Reference library generator
+    LIB_REFERENCE_ADDED = 'LIB_REFERENCE_ADDED'
```

Comment:
More recent changes in tools-suite should be applied here
```
    // ADT Tools - App generator
    ADT_TOOLS_APP_GEN = 'ADT_TOOLS_APP_GEN',
    // FIORI AI
    AI_GEN = 'AI_GEN'
```

Reply:
@devinea Thanks to bring this up. Actually we might consider remove the event name definition in open source version. Each consumer module defines its own set of events. Let me check with Ashely on this.